### DISCUSSION TITLE
Index jpg only

### DISCUSSION THREAD

**Discussion by smolesen** [photoprism/photoprism#3109] [Q&A]
Hi

In my library, I have both .jpg and .tif but I would like the indexer to only scan the .jpg files... is that possible?

TIA
/Søren

**Comment by lastzero** [comment]
:point_right:  https://docs.photoprism.app/user-guide/library/originals/#ignoring-files-and-folders

**Reply by lastzero** [reply]
Note, however, that PhotoPrism would create JPEG sidecar files for each TIFF file, even if you ignored JPEG files. Otherwise, it could neither index them nor create thumbnails. Ignoring TIFF files is fine though.

**Reply by smolesen** [reply]
Aha sorry, should have read through the doc more thoroughly....

/Søren
